src/main/java/com/mongodb/jdbc/logging/MongoLoggerUtils.java

```diff
@@ -0,0 +1,83 @@
+package com.mongodb.jdbc.logging;
+
+import com.mongodb.MongoException;
+import com.mongodb.jdbc.MongoConnection;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.*;
+
+public class MongoLoggerUtils {
+    private static ConcurrentHashMap<Integer, Logger> loggerPerConnection =
+            new ConcurrentHashMap<>();
```

Comment:
I removed MongoLoggerUtils altogether. This would indeed leak and it was old code from when I wanted an higher level parent logger if logging from MongoDriver.